### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]: Jaws and NVDA  announces additional inappropriate information when "Run on web app for containers" dialog   is opened 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1481]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Navigate to and select Project ->Azure->Run on web app for containers
4. "Run on web app for containers" dialog will open. 
5. Observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and  NVDA announces additional inappropriate information when "Run on web app for containers" dialog is opened 

**Jaws behaviour:**
 Run on Web App for Containers dialog
Invalid Data: Please specify Web App for Containers.Artifact Azure Container Registry Docker File Server URL Username Password Image and tag Container Registry Startup File wiprosample1.azurecr.io/ Web App for Containers Name Resource Group Before launch: Build Artifacts, Activate tool window 
Combo box closed
blank
1 of 1
To change the selection use the Arrow keys.

**NVDA Behaviour:**
Run on Web App for Containers  dialog  
Invalid Data: Please specify a valid docker file.
Artifact
Azure Container Registry
Docker File
Server URL
Username
Password
Image and tag
Container Registry
Startup File
 /
Web App for Containers
Before launch: Activate tool window
panel
combo box  collapsed
list
combo box  collapsed
list
combo box  collapsed
list
list item  selected  1 of 23
combo box  collapsed
list
combo box  collapsed
list
list item  selected  1 of 1

**Expected Result:**
Jaws and NVDA and NVDA should announce only the dialog name when dialog is opened .
i.e  Run on Web App for Containers dialog

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app for container-opened](https://user-images.githubusercontent.com/22956130/36105667-3eac5904-103b-11e8-8f1c-21a93e577b4f.jpg)
![62796447](https://user-images.githubusercontent.com/22956130/36105676-421d6dc6-103b-11e8-9706-c1c00654415c.jpg)

**Comment by Eskibear** [comment]
duplicated with #1485

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749910
